<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>605</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>605</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="6,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,2,0,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="mod_move_method_label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Перемещение модов:</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="before_game_start_rb">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Перед запуском БЛ</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">mod_move_method_group</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="on_switch_rb">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>При включении/выключении мода</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">mod_move_method_group</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="when_update_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Когда обновлять список модов:</string>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="on_launch_update_rb">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>При запуске менеджера</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">update_group</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="on_button_click_rb">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>По нажатию кнопки</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">update_group</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="path_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Пути для папок:</string>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="11,20,0">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="enabel_folder_label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Расположение модов:</string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <property name="indent">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enabel_folder_name">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browse_enabel_folder_button">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Выбрать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="11,20,0">
         <item>
          <widget class="QLabel" name="disabel_folder_label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Папка для выключения: </string>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <property name="indent">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="disabel_folder_name">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browse_disable_folder_button">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Выбрать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,5">
       <item>
        <widget class="QPushButton" name="open_settings_button">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Открыть папку с настройками</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="update_group"/>
  <buttongroup name="mod_move_method_group"/>
 </buttongroups>
</ui>
